### DISCUSSION TITLE
Navigation bar

### DISCUSSION THREAD

**Discussion by rafikrh** [Budibase/budibase#10073] [Help] [answered]
Hello everyone.
I want to add a clock (iframe) in this place.
I already have the iframe, I just want to know how to add it in this area.
![clock](https://user-images.githubusercontent.com/128312123/226216483-24dfd14c-5c86-4e89-aefe-4bed6fc7f1be.PNG)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @rafikrh,

To my knowledge you aren't able to edit this aside from changing the logo, app name, colours and navigation links. The best you'd be able to do is add an embed component in the screen and paste your Iframe code in there and align it to the right. (You may be able to play around with negative margins and other css properties in the configure css area in order to position it where you want.)

I hope this makes sense and answers your question.
